downloader/build.gradle

```diff
@@ -4,9 +4,61 @@ apply plugin: 'application'
 applicationName = 'downloader'
 mainClassName = 'com.here.ort.downloader.Main'
 
+sourceSets {
+    funTest {
+        kotlin.srcDirs 'src/funTest/kotlin'
+    }
+    intTest {
```

Comment:
As discussed internally, please use [kotlintest tags](https://github.com/kotlintest/kotlintest/blob/master/doc/reference.md#grouping-tests-with-tags) for functional tests instead of introducing a new integration test source set.